base-image/Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'fluent-plugin-amqp', "0.14.0"
 gem 'fluent-plugin-azure-loganalytics', "0.7.0"
 gem 'fluent-plugin-cloudwatch-logs', "0.8.0"
```

Comment:
@jliao2011 Can we also update fluent-plugin-cloudwatch-logs to latest version `v0.13.4` ?